docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
        - hmrc-postgres13:/var/lib/postgresql/data
   hmrc-opensearch:
     container_name: hmrc-opensearch
-    image: opensearchproject/opensearch:2
+    image: opensearchproject/opensearch:2.11.0
```

Comment:
I'd say either leave this out, or move it to its own PR since its not relevant to this PR

In general I'm inclined to say leave it as is, which forces someone to solve the env var question properly rather than just locking us to an outdated version